build/kubectl-testkube/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# syntax=docker/dockerfile:1
+ARG ALPINE_IMAGE
```

Comment:
I noticed that in the issue description you specified Ubuntu as a base image. Will alpine be sufficient in this case as well?